dask_sql/physical/rel/custom/wrappers.py

```diff
@@ -0,0 +1,497 @@
+# Copyright 2017, Dask developers
+# Dask-ML project - https://github.com/dask/dask-ml
+"""Meta-estimators for parallelizing estimators using the scikit-learn API."""
+import logging
+import warnings
+
+import dask.array as da
+import dask.dataframe as dd
+import dask.delayed
+import numpy as np
+
+try:
+    import sklearn.base
+    import sklearn.metrics
+except ImportError:  # pragma: no cover
+    raise ValueError("sklearn must be installed")
```

Comment:
Minor nit: I would prefer the raised error here to be an `importError` as well. 
Looking through the codebase it seems like the original errors were also `valueErrors` but `importError` might be cleaner